src/hooks/chat/useGetMyChatList.ts

```diff
@@ -0,0 +1,14 @@
+import { getChatList } from '@/apis/chat';
+import { CHAT_KEYS } from '@/constants/@queryKeys';
+import { useQuery } from '@tanstack/react-query';
+
+const useGetMyChatList = () => {
+  const { data } = useQuery({
+    queryKey: CHAT_KEYS.CHAT_LIST(),
+    queryFn: getChatList,
+  });
+
+  return { data };
```

Comment:
```suggestion
  return useQuery({
    queryKey: CHAT_KEYS.CHAT_LIST(),
    queryFn: getChatList,
  });
```
이렇게 바로 return하는 방법도 있을 것 같은데 혹시 나중에 더 return할 값이 생길 걸 고려하신걸까요?? (useGetSecondaryToken 동일)

Reply:
useEnterChatRoom를 만들고 난 뒤에 만들어서, 습관적으로 return을 `{ data }`로 만들었던 것 같습니다. 수정했습니다!